packages/pigment-css-react-new/package.json

```diff
@@ -0,0 +1,152 @@
+{
+  "name": "@pigment-css/react-new",
```

Comment:
Hey there. Thanks for your work. A question about this package name, are we going to remove the `-new`  before it is released?